frontend/src/metabase/search/containers/SearchApp.jsx

```diff
@@ -91,11 +91,7 @@ function SearchApp({ location }) {
                   <EmptyState
                     title={t`Didn't find anything`}
                     message={t`There weren't any results for your search.`}
-                    illustrationElement={
-                      <Box mb={"-2.5rem"}>
-                        <img src={NoResults} />
-                      </Box>
-                    }
+                    illustrationElement={<NoObjectError mb="-1.5rem" />}
```

Comment:
Reduce the negative bottom margin by `1rem` (-2.5rem -. -1.5rem) otherwise it will be too close to the label
![image](https://github.com/metabase/metabase/assets/1937582/8ea41134-7702-4519-85fd-27b118d24585)